recipes/xerces-c/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [ `uname` == Darwin ]; then
+	export CC=clang
+	export CXX=clang++
```

Comment:
@groutr, these are :+1: in my opinion. Let's see what @ocefpaf says to my explanation above.